src/library/components/index.js

```diff
@@ -0,0 +1,17 @@
+// Index for all Components: import { AppBar, FAB } from 'library/components'
+
+import AppBar from './AppBar';
+import FAB from './FAB';
+import IconButton from './IconButton';
+import Menu from './Menu/MenuContainer';
+import MenuDivider from './Menu/MenuDivider';
+import MenuItem from './Menu/MenuItem';
+
+export {
+  AppBar,
+  FAB,
+  IconButton,
+  Menu,
+  MenuDivider,
+  MenuItem,
+}
```

Comment:
✏️
inspiration from https://stackoverflow.com/questions/34072598/es6-exporting-importing-in-index-file
```suggestion
export {default as AppBar} from './AppBar';
export {default as FAB} from './FAB';
export {default as IconButton} from './IconButton';
export {default as Menu} from './Menu/MenuContainer';
export {default as MenuDivider} from './Menu/MenuDivider';
export {default as MenuItem} from './Menu/MenuItem';
```

Reply:
Ha this was actually on my to-do list. I found out about it a few days ago when I was on the ecmascript website